index.d.ts

```diff
@@ -7,16 +7,23 @@ declare namespace pTimeout {
 	type TimeoutError = TimeoutErrorClass;
 }
 
+/**
+ * Promise with `.clear() method
```

Comment:
This is not useful. It just describes what the types already clearly define. The doc comment should describe what it does and its purpose for humans.

Reply:
It's also in the incorrect place.

Reply:
Hi, what about `Promise with .clear() method that clears the timeout.`? 
And i didn't understand why it is in the incorrect place. It is just on top of interface declaration.

Reply:
The doc comment should be placed above the `clear: ()` and should describe what **`.clear()` does.